docs/sphinx/developer_guide.rst

```diff
@@ -77,3 +80,30 @@ It is also possible to use the CI script outside of CI:
 .. code-block:: bash
 
   $ HOST_CONFIG=<path_to>/<host-config>.cmake scripts/gitlab/build_and_test.sh
+
+Using Uberenv to configure and run Leak Sanitizer
+-------------------------------------------------
+
+During development, it may be beneficial to regularly check for memory leaks. This will help avoid the possibility of having many memory leaks showing up all at once during the CI tests later on. The Leak Sanitizer can easily be configured from the root directory with:
+
+.. code-block:: bash
+
+  $ srun -ppdebug -N1 --exclusive python scripts/uberenv/uberenv.py --spec="%clang@9.0.0 cxxflags=-fsanitize=address"
+  $ cd build
+  $ cmake -C <path_to>/hc-quartz-toss_3_x86_64_ib-clang@9.0.0.cmake ..
+  $ cmake --build -j
+  $ ASAN_OPTIONS=detect_leaks=1 make test
+  
+This will configure a build with Clang 9.0.0 and the Leak Sanitizer. If there is a leak in one of the tests, it can be useful to gather more information about what happened and more details about where it happened. One way to do this is to run:
```

Comment:
Could you add a note saying that the CMake file `hc-quartz-....` can be re-used?